react/features/base/toolbox/components/ToolboxItem.web.js

```diff
@@ -48,25 +65,13 @@ export default class ToolboxItem extends AbstractToolboxItem<Props> {
      * @returns {ReactElement}
      */
     _renderIcon() {
-        const { iconName, tooltipPosition, showLabel } = this.props;
+        const { iconName, showLabel } = this.props;
```

Comment:
with the simplification here you can just return the result of `React.createElement`.